admin-portal/src/components/sidebar.tsx

```diff
@@ -0,0 +1,75 @@
+"use client";
+
```

Comment:
This directive marks a page as a client component instead of the nextJS default which is server components (loaded in server and a static file returned to client). 

This had to be added so that I could use `usePathname` which is allowed only on Client components